agent/agentssh/agentssh_test.go

```diff
@@ -200,7 +201,11 @@ func TestNewServer_CloseActiveConnections(t *testing.T) {
 				}
 				assert.NoError(t, err)
 
+				// Allow the session to settle (i.e. reach echo).
 				pty.ExpectMatchContext(ctx, "started")
+				// Sleep a bit to ensure the sleep has started.
+				time.Sleep(testutil.IntervalMedium)
```

Comment:
nit: maybe we should only sleep on `runtime.GOOS == "darwin"`?

Reply:
I kind of agree, but also worry about this being a very hard to trigger edge case on other OS's as well :/.